client/bootstrap.js

```diff
@@ -0,0 +1,17 @@
+window.module = {}
```

Comment:
I wonder if this is still needed with the latest webpack, it might not be needed anymore

Reply:
Removing it breaks everything.

Reply:
Fixed here: https://github.com/zeit/next.js/pull/5093